apps/server/src/shared/domain/entity/school.entity.ts

```diff
@@ -106,6 +98,15 @@ export class SchoolEntity extends BaseEntity {
 	@ManyToOne(() => FederalStateEntity, { fieldName: 'federalState', nullable: false })
 	federalState: FederalStateEntity;
 
+	@Property({ nullable: true })
+	county?: County;
+
+	@Property({ nullable: true })
+	purpose?: SchoolPurpose;
+
+	@Property({ nullable: true })
+	enableStudentTeamCreation?: boolean;
```

Comment:
Die anderen Props vom Moduel die alle irgendwas Richtung Features und Purpose beinhalten sind hier nicht drin. Brauchen wir die noch? Die könnten ja alle Richtung purpose und features gemapped werden beim Domain Object. Aber nur wenn es irgendwo in den Clients / Nest benötigt wird.

Reply:
Ich hab die Prop auf ein Feature im DO gemappt, siehe den SchoolEntityMapper.